commons/util/src/main/java/io/github/microcks/util/graphql/GraphQLSchemaValidator.java

```diff
@@ -85,14 +84,7 @@ public static JsonNode buildResponseJsonSchema(String schemaText, String query)
     * @return The list of validation failures. If empty, json object is valid !
     */
    public static List<String> validateJson(JsonNode schemaNode, JsonNode jsonNode) {
-      try {
-         return JsonSchemaValidator.validateJson(schemaNode, jsonNode);
-      } catch (ProcessingException e) {
-         log.debug("Got a ProcessingException while trying to interpret schemaNode as a real schema");
-         List<String> errors = new ArrayList<>();
-         errors.add("schemaNode does not seem to represent a valid Json schema");
-         return errors;
-      }
+      return JsonSchemaValidator.validateJson(schemaNode, jsonNode);
```

Comment:
Is the "invalid schemaNode" error included in `JsonSchemaValidator.validateJson()` now?